packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -101,6 +107,14 @@ class MemoryBodyState extends State<MemoryBody>
   void initState() {
     super.initState();
 
+    eventChartController = ChartController(
+      displayYLabels: false,
+      displayXAxis: false,
+      displayXLabels: false,
+    );
+    vMChartController = ChartController(displayTopLine: false);
```

Comment:
vmChartController not vMChartController.

Reply:
Done.